src/views/dmca/dmca.jsx

```diff
@@ -0,0 +1,32 @@
+var React = require('react');
+var FormattedMessage = require('react-intl').FormattedMessage;
+var render = require('../../lib/render.jsx');
+var Page = require('../../components/page/www/page.jsx');
+var Box = require('../../components/box/box.jsx');
+
+require('./dmca.scss');
+
+var Dmca = React.createClass({
+    type: 'Dmca',
+    render: function () {
+        return (
+            <div className="inner">
+                <Box title={'DMCA'}>
+                    <p><FormattedMessage id='dmca.intro' /></p>
+                    <p>Copyright Agent / Mitchel Resnick<br/>
+                    MIT Media Laboratory<br/>
+                    77 Massachusetts Ave<br/>
+                    Room E14-445A<br/>
+                    Cambridge, MA 02139<br/>
+                    Tel: (617) 253-9783</p>
```

Comment:
Just one nitpick (sorry) – our convention for tabbing with multiline paragraphs is to indent the text on a new line, like this:
```
<p>
    Copyright Agent / Mitchel Resnick<br/>
    MIT Media Laboratory<br/>
    77 Massachusetts Ave<br/>
    Room E14-445A<br/>
    Cambridge, MA 02139<br/>
    Tel: (617) 253-9783
</p>
```
If you could change that, I think we're all set, because looking at it in the browser it looks great!